src/node_revert.h

```diff
@@ -15,18 +16,16 @@
  **/
 namespace node {
 
-#define SECURITY_REVERSIONS(XX)
-//  XX(CVE_2016_PEND, "CVE-2016-PEND", "Vulnerability Title")
+#define SECURITY_REVERSIONS(XX)   \
+  XX(CVE_2016_PEND, "CVE-2016-PEND", "Vulnerability Title")
```

Comment:
Is this changed intentionally?

Reply:
nope, changed it to test and forgot to change it back